projects/plugins/jetpack/modules/widgets/upcoming-events.php

```diff
@@ -26,106 +50,140 @@ function css() {
 	display: block;
 }
 </style>
-<?php
+		<?php
 	}
 
-	function form( $instance ) {
+	/**
+	 * Displays the form for this widget on the Widgets page of the WP Admin area.
+	 *
+	 * @param array $instance Instance configuration.
+	 *
+	 * @return void
+	 */
+	public function form( $instance ) {
 		$defaults = array(
 			'title'    => __( 'Upcoming Events', 'jetpack' ),
 			'feed-url' => '',
 			'count'    => 3,
 		);
 		$instance = array_merge( $defaults, (array) $instance );
-?>
+		?>
 
 		<p>
-		<label for="<?php echo $this->get_field_id( 'title' ); ?>"><?php _e( 'Title:', 'jetpack' ); ?></label>
-		<input class="widefat" id="<?php echo $this->get_field_id( 'title' ); ?>" name="<?php echo $this->get_field_name( 'title' ); ?>" type="text" value="<?php echo esc_attr( $instance['title'] ); ?>" />
+		<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>"><?php esc_html_e( 'Title:', 'jetpack' ); ?></label>
+		<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>" type="text" value="<?php echo esc_attr( $instance['title'] ); ?>" />
 		</p>
 
 		<p>
-		<label for="<?php echo $this->get_field_id( 'feed-url' ); ?>"><?php _e( 'iCalendar Feed URL:', 'jetpack' ); ?></label>
-		<input class="widefat" id="<?php echo $this->get_field_id( 'feed-url' ); ?>" name="<?php echo $this->get_field_name( 'feed-url' ); ?>" type="text" value="<?php echo esc_attr( $instance['feed-url'] ); ?>" />
+		<label for="<?php echo esc_attr( $this->get_field_id( 'feed-url' ) ); ?>"><?php esc_html_e( 'iCalendar Feed URL:', 'jetpack' ); ?></label>
+		<input class="widefat" id="<?php echo esc_attr( $this->get_field_id( 'feed-url' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'feed-url' ) ); ?>" type="text" value="<?php echo esc_attr( $instance['feed-url'] ); ?>" />
 		</p>
 
 		<p>
-		<label for="<?php echo $this->get_field_id( 'count' ); ?>"><?php _e( 'Items to show:', 'jetpack' ); ?></label>
-		<select id="<?php echo $this->get_field_id( 'count' ); ?>" name="<?php echo $this->get_field_name( 'count' ); ?>">
+		<label for="<?php echo esc_attr( $this->get_field_id( 'count' ) ); ?>"><?php esc_html_e( 'Items to show:', 'jetpack' ); ?></label>
+		<select id="<?php echo esc_attr( $this->get_field_id( 'count' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'count' ) ); ?>">
 			<?php
 			$i = 1;
 			while ( $i <= 10 ) {
-			?>
-				<option <?php selected( $instance['count'], $i ); ?>><?php echo $i; ?></option>
+				?>
+				<option <?php selected( $instance['count'], $i ); ?>><?php echo (int) $i; ?></option>
 			<?php $i++; } ?>
```

Comment:
Seems weird that this isn't a for loop.
```suggestion
			<?php for ( $i = 1; $i <= 10; $i++ ) { ?>
				<option <?php selected( $instance['count'], $i ); ?>><?php echo (int) $i; ?></option>
			<?php } ?>
```

Reply:
👍 75bdcd8c991a1da7e475fe980f62c24cdcae3652